src/store/modules/keycodes/index.js

```diff
@@ -27,17 +31,58 @@ const state = {
   active: 'ANSI'
 };
 
-function isISO() {
-  return store.state.app.configuratorSettings.iso;
+function getOSKeyboardLayout() {
+  const fallbackOSKeyboardLayout = 'keymap_us';
+  const osKeyboardLayout =
+    store.state.app.configuratorSettings.osKeyboardLayout;
+  if (
+    isUndefined(osKeyboardLayout) ||
+    !Object.keys(keymapExtras).includes(osKeyboardLayout)
+  ) {
+    console.log(
+      `The stored OS keyboard layout value (${osKeyboardLayout}) is not a valid value! Falling back to '${fallbackOSKeyboardLayout}'.`
+    );
+    store.state.app.configuratorSettings.osKeyboardLayout =
+      fallbackOSKeyboardLayout;
+  }
+  return store.state.app.configuratorSettings.osKeyboardLayout;
+}
+
+function isANSI() {
+  return keymapExtras[getOSKeyboardLayout()].isANSI;
+}
+
+function toLocaleKeycode(keycodeLUT, keycodeObject) {
+  console.assert(!isUndefined(keycodeLUT));
+  if (
+    !Object.keys(keycodeObject).includes('name') ||
+    !Object.keys(keycodeObject).includes('code')
+  ) {
+    // Not an object describing a keyboard key; return as is
+    return keycodeObject;
+  }
+  if (keycodeLUT[keycodeObject.code]) {
+    // Clone in a shallow manner the original keycodeObject object and
+    // override the name, title, and possibly other fields
+    return { ...keycodeObject, ...keycodeLUT[keycodeObject.code] };
+  } else {
+    return keycodeObject;
+  }
 }
 
-function generateKeycodes(isIso, isSteno) {
+function generateKeycodes(osKeyboardLayout, isSteno) {
+  store.state.app.configuratorSettings.iso = !isANSI();
```

Comment:
same here. Use an update to configurator settings, don't change it like this directly. Cross store updates like this is buggy and untracked by vuex.

Reply:
Solved in 2f5c48a